Dockerfile

```diff
@@ -62,8 +62,9 @@ RUN apk add --no-cache --virtual .rubymakedepends \
     asciimath \
     "asciidoctor-pdf:${ASCIIDOCTOR_PDF_VERSION}" \
     "asciidoctor-revealjs:${ASCIIDOCTOR_REVEALJS_VERSION}" \
+    bigdecimal \
```

Comment:
If bigdecimal is needed, install the version shipped with ruby, it's provided by Alpine package ruby-bigdecimal.